src/models.js

```diff
@@ -8,13 +8,100 @@
 import moment from 'moment';
 import { t } from 'ttag';
 
+/**
+ * Map a version name to its corresponding type;
+ *
+ * @enum
+ * @readonly
+ */
+const VersionType = {
+  blockVersion: 0,
+  transactionVersion: 1,
+  createTokenTransactionVersion: 2,
+  mergedMinedBlockVersion: 3,
+  nanoContractTransaction: 4,
+};
+
+/**
+ * Map version type to info that should assist on token list visualization.
+ *
+ * @readonly
+ */
+const VersionInfo = {
+  [VersionType.blockVersion]: { symbol: 'b', label: t`Block` },
+  [VersionType.transactionVersion]: { symbol: 'tx', label: t`Transaction` },
+  [VersionType.createTokenTransactionVersion]: { symbol: 'tk', label: t`Create Token Transaction` },
+  [VersionType.mergedMinedBlockVersion]: { symbol: 'mb', label: t`Merged Mined Block` },
+  [VersionType.nanoContractTransaction]: { symbol: 'nc', label: t`Nano Contract Transaction` },
+};
+
```

Comment:
I think we should use the method we already have in the lib: https://github.com/HathorNetwork/hathor-wallet-lib/blob/master/src/utils/transaction.ts#L629

The explorer and desktop wallet use it, and it's better to have a single source of information, so it's easier to maintain.

Reply:
I need a map to the translation, the lib doesn't give me a translation.

Reply:
We have many places where we get strings from the lib, e.g., error messages that are not translated. 

We already discussed ways of handling this. I don't think the best way is to duplicate what we have in the lib, this decreases code maintainability a lot.

For now, I prefer keeping these strings not translated and getting them from a single source of information for this kind of thing, which is the lib method.

We need to discuss this in an issue for sure, can you create one for this?

Reply:
Sure! I will open an issue for this.

Reply:
Here the issue: https://github.com/HathorNetwork/internal-issues/issues/292

Reply:
Removed translation mapping: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/434/commits/ccd0f1fad0300196851d1c2287829e8600789d1d